collective/googlenews/browser/templates/atom.xml.pt

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" ?>
```

Comment:
@rodfersou can we avoid having to include a copy of this template? can we use Plone default template?